### PR TITLE
SCADAfence integration (#1910)

### PR DESCRIPTION
**Pull request by hirshbergshachar**
Integrations to provide access to SCADAfence CNM data for:
getting alerts
getting asset details
getting asset connections
getting asset traffic statistics
updating alerts status in SCADAfence CNM

### CODE REVIEW

File: Integrations/integration-SCADAfence_CNM.yml [hunk 1]
```diff
@@ -0,0 +1,467 @@
...
+  type: 4
+  required: true
+- display: API auth key
+  name: APIKey
+  defaultvalue: 
+  type: 4
+  required: true
+- display: API url
+  name: APIUrl
+  defaultvalue: https://172.28.128.10/externalApi
```
Comment (anara123) [inline Integrations/integration-SCADAfence_CNM.yml]: why not to add `externalApi` in the integration itself?

File: Integrations/integration-SCADAfence_CNM.yml [hunk 2]
```diff
@@ -0,0 +1,467 @@
...
+        get_alerts('Critical', None)
+        demisto.results('ok')
+        sys.exit(0)
+
+    elif demisto.command() == 'scadafence-getAlerts':
+
+        data = get_alerts(demisto.args()['severity'], None)
+        md = '## Following alerts were found\n'
+        md += tableToMarkdown('Alerts are: ', data)
+        # tableToMarkdown('header', [{}, {}, {}...])
```
Comment (anara123) [inline Integrations/integration-SCADAfence_CNM.yml]: remove this comment

File: Integrations/integration-SCADAfence_CNM.yml (hunk 1 above)
Comment (hirshbergshachar) [reply Integrations/integration-SCADAfence_CNM.yml]: @AlexzxelA is the `externalApi` path constant? If so, perhaps we can set only the server URL here.

Comment (AlexzxelA) [reply Integrations/integration-SCADAfence_CNM.yml]: it is sure possible

Comment (hirshbergshachar) [reply Integrations/integration-SCADAfence_CNM.yml]: @anara123 @AlexzxelA concatenating it to the API url now.

**Comment by hirshbergshachar** [comment]
@anara123 we have test env in place, all good here.

**Pull request merged**